include/megaapi_impl.h

```diff
@@ -1547,6 +1548,44 @@ class MegaUserAlertListPrivate : public MegaUserAlertList
     int s;
 };
 
+class MegaRecentActionBucketPrivate : public MegaRecentActionBucket
+{
+public:
+    MegaRecentActionBucketPrivate(MegaClient::recentaction& ra, MegaClient* mc);
+    MegaRecentActionBucketPrivate(int64_t timestamp, const string& user, handle parent, bool update, bool media, MegaNodeList*);
+    virtual ~MegaRecentActionBucketPrivate();
+    virtual MegaRecentActionBucket *copy() const;
+    virtual int64_t getTimestamp() const;
+    virtual const char* getUserEmail() const;
+    virtual MegaHandle getParentHandle() const;
+    virtual bool getIsUpdate() const;
+    virtual bool getIsMedia() const;
+    MegaNodeList* getNodes();
+
+private:
+    int64_t timestamp;
+    string user;
+    handle parent;
+    bool update, media;
+    MegaNodeList* list;
```

Comment:
A meaningful name is always a plus. Better to name it `nodes` here

Reply:
cool